core/jobs/batch_jobs/exploration_migration_jobs.py

```diff
@@ -0,0 +1,423 @@
+            | 'Filter oks' >> beam.Filter(
+                lambda result_item: result_item.is_ok())
+            | 'Unwrap ok' >> beam.Map(
+                lambda result_item: result_item.unwrap())
+        )
+
+        migrated_exploration_summary_results = (
+            exploration_summary_models
+            | 'Transform and migrate exploration summary model' >> beam.MapTuple( # pylint: disable=line-too-long
+                self._migrate_exploration_summary)
+        )
+        migrated_exploration_summaries = (
+            migrated_exploration_summary_results
+            | 'Filter ok' >> beam.Filter(
+                lambda result_item: result_item.is_ok())
+            | 'Unwrap oks' >> beam.Map(
+                lambda result_item: result_item.unwrap())
+        )
+
+        migrated_exploration_job_run_results = (
+            migrated_exploration_results
+            | 'Generate results for migration' >> (
+                job_result_transforms.ResultsToJobRunResults(
+                    'EXPLORATION PROCESSED'))
+        )
+        migrated_exploration_summary_job_run_results = (
+            migrated_exploration_summary_results
+            | 'Generate results for summary migration' >> (
+                job_result_transforms.ResultsToJobRunResults(
+                    'EXPLORATION SUMMARY PROCESSED'))
+        )
+
+        exploration_changes = (
+            unmigrated_exploration_models
+            | 'Generate exploration changes' >> beam.FlatMapTuple(
+                self._generate_exploration_changes)
+        )
+
+        exploration_objects_list = (
+            {
+                'exp_model': unmigrated_exploration_models,
+                'exploration_summary_model': exploration_summary_models,
+                'exploration': migrated_explorations,
+                'exploration_summary': migrated_exploration_summaries,
+                'exploration_changes': exploration_changes,
+                'exploration_rights_model': exploration_rights_models
+            }
+            | 'Merge objects' >> beam.CoGroupByKey()
+            | 'Get rid of ID' >> beam.Values()  # pylint: disable=no-value-for-parameter
+            | 'Remove unmigrated exploration' >> beam.Filter(
+                lambda x: len(x['exploration_changes']) > 0 and len(x['exploration']) > 0 and len(x['exploration_summary']) > 0) # pylint: disable=line-too-long
+            | 'Reorganize the exploration objects' >> beam.Map(lambda objects: {
+                    'exp_model': objects['exp_model'][0],
+                    'exploration_summary_model': (
+                        objects['exploration_summary_model'][0]),
+                    'exploration': objects['exploration'][0],
+                    'exploration_summary': objects['exploration_summary'][0],
+                    'exploration_changes': objects['exploration_changes'],
+                    'exploration_rights_model': (
+                        objects['exploration_rights_model'])
```

Comment:
```suggestion
                    'exploration_rights_model': (
                        objects['exploration_rights_model'][0])
```

and down below you don't need to do this

Reply:
Fixed